src/components/AdminSettings/MatterbridgeIntegration.vue

```diff
@@ -138,12 +152,13 @@ export default {
 				this.matterbridgeVersion = response.data.ocs.data.version
 				this.matterbridgeEnabled = true
 				this.saveMatterbridgeEnabled()
-
-				this.isInstalling = false
+				this.error = ''
 			} catch (e) {
 				console.error(e)
 				showError(t('spreed', 'Failed to execute Matterbridge binary.'))
+				this.error = e.response.data.ocs.data.error
```

Comment:
There could be an error where this is undefined (e.g. maintenance mode). So it would need some sanity check and a fallback error message I guess?